cli/script/command-executor.ts

```diff
@@ -59,8 +62,12 @@ export interface PackageWithMetrics {
     metrics?: UpdateMetricsWithTotalActive;
 }
 
-export var sdk: AccountManager;
 export var log = (message: string | Chalk.ChalkChain): void => console.log(message);
+export var sdk: AccountManager;
+export var spawn = childProcess.spawn;
+export var getTmpDir = (): string => {
```

Comment:
Ah OK. It might be worth looking into Jest as a test runner at some point. It auto mocks required modules (with the ability to fake their behavior), which makes it pretty clean to write unit tests, without needing to abstract existing modules any further simply to be able to mock/stub them within tests.